changelog/233.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:docs
```

Comment:
There isn't a section in the template for a note type of `docs` so this would not show up in the changelog. We could add it to the template, or else we could make this a note type that will be in the changelog, such as `improvement`. Rather than changing the template, though, I think just calling it an improvement would be best.

```suggestion
```release-note:improvement
```